app/views/devise/registrations/_registration_form.html.erb

```diff
@@ -1,60 +1,65 @@
-<div class="new-article-pitch">
-  <% if @new_article_not_logged_in %>
-    <div class="prefill-non-logged-in-message">
-      <div class="registration__logo-container">
-        <img class="secondary-logo" src="<%= optimized_image_url(safe_logo_url(SiteConfig.secondary_logo_url), width: 190) %>"
-             alt="Secondary <%= community_name %> logo" />
-      </div>
-      <p>
-        <b>Welcome to <%= community_name %></b>
-      </p>
-      <p>
-        Sign in below to compose your post and share it with the community.
+<section class="crayons-layout">
+  <div class="registration crayons-card">
+    <div class="registration__content">
+      <h1 class="registration__title">
+        Welcome to <%= community_name %>
+      </h1>
+      <p class="registration__description">
+        <a href="/"><%= community_name %></a> is a community of <%= number_with_delimiter User.registered.estimated_count %> amazing <%= community_members_label %>.
```

Comment:
Suggestion: Is there a controller involved in this flow that we could use to pass in the value of `User.registered.estimated_count`? Direct model access in a view can sometimes be a code smell (I'm not saying it is here however!)